packages/e2e/cypress/support/commands.ts

```diff
@@ -73,8 +73,7 @@ Cypress.Commands.add('findInputField', (field: string) => {
   const regex = new RegExp(regexString, 'i');
 
   if (isPasswordField) {
-    // TODO: we should use cy.findByLabelText once our dom is cleaned up
-    return cy.findByPlaceholderText(regex);
+    return cy.findByLabelText(field);
```

Comment:
@eddiekeller @slaymance Updated this since `findByLabelText` should be working now.